src/assets/scripts/mededelingen/MededelingDetail.jsx

```diff
@@ -43,7 +45,7 @@ class MededelingDetail extends React.Component {
     if(mededeling) {
       return <div>
         <h1>{mededeling.titel}</h1>
-
+        <ProfielLink uid={parseInt(mededeling.user)} />
```

Comment:
remove parseInt, and use `pk` instead of uid for the name (see `docs/conventions.md`). user pks are not ints but 4 chars.